libs/core/calendar/calendar-views/calendar-month-view/calendar-month-view.component.ts

```diff
@@ -213,16 +262,183 @@ export class CalendarMonthViewComponent<D> implements OnInit, OnChanges, Focusab
         return id + this._monthOffset === this.monthSelected;
     }
 
+    /** Method for handling the mouse click event when a month is selected  */
+    selectMonth(monthCell: CalendarMonth<D>, event?: MouseEvent): void {
+        if (event) {
+            event.stopPropagation();
+            event.preventDefault();
+        }
+        this.monthSelected = monthCell.month;
+        this.monthClicked.emit(this.monthSelected);
+
+        // when date range start date is selected start the hover effect between the date range
+        if (this.isDateRangeMonthYearFormat) {
+            this._selectRangeDate(monthCell);
+            this._handleRangeHoverEffect(event);
+        }
+    }
+
+    /**
+     * Set focus on month cell.
+     * It can be a selected cell, current month cell or first cell in the list
+     */
+    setFocusOnCell(): void {
+        const cellToFocus = new DefaultCalendarActiveCellStrategy().getActiveCell(this._getMonthList());
+        if (!cellToFocus?.id) {
+            return;
+        }
+        this._focusElementBySelector(`#${cellToFocus.id}`);
+    }
+
+    /**
+     * @hidden
+     * Handles hover effect for single-range selection mode.
```

Comment:
What is "single-range"? I think this should just be "range"?

Reply:
@mikerodonnell89 corrected